three_ANA_sites/siteInfo.csv

```diff
@@ -1,10 +1,10 @@
-station,station_name,latitude,longitude,drainage_area,constituent,constit_units
-RONC02800,Ronc,1,1,100,NO3,mg L^-1
-RONC02800,Ronc,1,1,100,PT,mg L^-1
-RONC02800,Ronc,1,1,100,Q,cms
-MOGU02900,Mogu,2,2,200,NO3,mg L^-1
-MOGU02900,Mogu,2,2,200,PT,mg L^-1
-MOGU02900,Mogu,2,2,200,Q,cms
-ORIZ02900,Oriz,3,3,300,NO3,mg L^-1
-ORIZ02900,Oriz,3,3,300,PT,mg L^-1
-ORIZ02900,Oriz,3,3,300,Q,cms
\ No newline at end of file
+file,site.id,site.name,lat,lon,basin.area,constituent,units
```

Comment:
reflections:

I think we're worrying here about saving the user a relatively small amount of typing. In one way, that's really important - redundancy means opportunities for error. In another way, not so much - these users are probably quite good at manually checking their input files, and they might even be good at automatically generating them.

The other thing we're discussing, which is definitely important, is clarity. Column names help with this. Text documentation will also help. In that vein, I changed the column names to match (more closely, anyway) the arguments in ?metadata to (1) make some text documentation immediately available, and (2) make it an easier transition in case users want to switch back and forth between batch mode and single-model mode. And I think you're on the right track here with trying to avoid confusion between different types of units.

The fully flat option has the merit of being exactly what we end up needing and producing within the main loop in batch.R. If you're keen to try out a multi-table approach that minimizes redundancy and maximizes clarity, and if you can try it out quickly, let's do it. I do think we should plan to fall back to a completely flat table with redundant info if the other options don't work well right away.